### PR TITLE
debugging to run w/ web-app

### PR DESCRIPTION
**Pull request by adewyer**

### CODE REVIEW

**Comment by adewyer** [comment]
- To run with the web-app the `"react-app"` section of `.babelrc` needs to be removed. It causes errors with transpiling code. 
- Manual installation of code using `npm install https://github.com/Exabyte-io/material-designer/#{commit hash}` allows for web-app to run.

File: .babelrc [hunk 1]
```diff
@@ -10,12 +9,6 @@
             }
         ],
         "@babel/preset-react",
-        [
-            "react-app",
-            {
-                "absoluteRuntime": false
-            }
-        ]
```
Comment (adewyer) [inline .babelrc:18]: react-app caused error w/ transpile preventing installation of code within `dist` directory

File: README.md [hunk 2]
```diff
@@ -74,6 +74,15 @@ sh run-tests.sh                                                 # to run all the
...
 
+## Running development Materials Designer Code with the Web-App
+
+Inside the `web-app/src/exabyte/package.json` file update the dependency for "@exabyte-io/materials-designer" to the following
+
+```bash
+"@exabyte-io/materials-designer": "https://github.com/Exabyte-io/materials-designer/#{development commit hash}",
+```
+Then launch the web-app from `exabyte-stack/web-app` directory using the `run-meteor.sh` script
+
```
Comment (timurbazhirov) [inline README.md:85]: This content should go to the web-app repository

File: .babelrc (hunk 1 above)
Comment (adewyer) [reply .babelrc:18]: this file is the same as `SOF-5238`, but it is not showing them as the same.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**